core/src/main/java/org/apache/calcite/avatica/remote/looker/LookerResponseParser.java

```diff
@@ -114,6 +122,25 @@ static Object deserializeValue(JsonParser parser, ColumnMetaData columnMetaData)
     }
   }
 
+  /**
+   * Collects the values of an array whose elements are represented by {@code metaData.type.rep}. Does
+   * not support nested arrays.
+   *
+   * @param metaData the {@link ColumnMetaData} for this value.
+   * @param parser a JsonParser whose current token is JsonToken.START_ARRAY. The parser is advanced
+   *        through the elements until the end of the array is reached. Parse does not advance past
+   *        END_ARRAY.
+   * @return An array of values with element type matching {@code metaData.type.rep}.
+   */
+  static Object deserializeArray(JsonParser parser, ColumnMetaData metaData) throws IOException {
```

Comment:
Why does it return type `Object`?

Reply:
Wasn't thinking to hard about it since it was being stuffed in an an array of objects anyway. 😅